index.d.ts

```diff
@@ -51,6 +51,46 @@ export type Instance = {
 
 export type Unmount = () => void;
 
+/**
+ * This hook is used for handling user input.
+ * It's a more convienient alternative to using `StdinContext` and listening to `data` events.
+ * The callback you pass to `useInput` is called for each character when user enters any input.
+ * However, if user pastes text and it's more than one character, the callback will be called only once and the whole string will be passed as `input`.
+ *
+ * ```js
```

Comment:
It's TS, not JS, but it's also implied, so no language tag needed.

```suggestion
 * ```
```